src/js/game/time/game_time.js

```diff
@@ -185,6 +190,19 @@ export class GameTime extends BasicSerializableObject {
         this.speed = speed;
     }
 
+    togglePause() {
+        if (this.getIsPaused()) {

```

Comment:
I wouldn't do that many checks here, just do `this.setSpeed(new RegularGameSpeed())`

Reply:
I made it like this so there can be other gamespeeds then just regular speed